patches/boost_context_asm_arm_syntax_unified.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/libs/context/src/asm/jump_arm_aapcs_elf_gas.S b/src/Boost/libs/context/src/asm/jump_arm_aapcs_elf_gas.S
+index d0f7fa2..58d11b0 100644
+--- a/libs/context/src/asm/jump_arm_aapcs_elf_gas.S
++++ b/libs/context/src/asm/jump_arm_aapcs_elf_gas.S
+@@ -42,6 +42,7 @@
+ .globl jump_fcontext
+ .align 2
+ .type jump_fcontext,%function
++.syntax unified
```

Comment:
note for posterity, the patch has been upstreamed in https://github.com/boostorg/context/pull/94

Reply:
I know, it was my pull request that got added, but ceph uses boost version 1.67.0 that is 2 releases behind ;-)

Reply:
thanks @the78mole. i was aware of this. this note is just for whomever interested in this change in future. for instance, they might want to learn the copyright of this patch or want to cleanup it when reading this change.